tests/manage/noobaa/test_bucket_deletion.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import pytest
+
+from tests.helpers import create_unique_resource_name
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import tier1
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def create_buckets(request, noobaa_obj):
+    """
+    Creates multiple buckets
+    """
+    created_buckets = []
+
+    bucket_name = create_unique_resource_name(
+        resource_description='bucket', resource_type='s3'
+    )
+    logger.info(f'Creating bucket: {bucket_name}')
+    created_buckets.append(
+        noobaa_obj.s3_create_bucket(bucketname=bucket_name)
+    )
+
+    def verify_bucket():
+        """
+        Verifies whether buckets exists after deletion
+
+        """
+        for bucket in created_buckets:
+            logger.info(f"Verifying whether bucket: {bucket.name} exists"
+                        f" after deletion")
+            noobaa_obj.s3_verify_bucket_exists(bucket)
```

Comment:
I think it'll be a good idea to add an assertion + return False from the method
```suggestion
            assert noobaa_obj.s3_verify_bucket_exists(bucket)
```

Reply:
This is outdated, was part of my first commit. I had made this change in my earlier commit.